operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -148,37 +148,63 @@ func (c *CentralComponentSpec) GetAdminPasswordGenerationDisabled() bool {
 	return pointer.BoolPtrDerefOr(c.AdminPasswordGenerationDisabled, false)
 }
 
-// CentralDBEnabled returns a bool if CentralDBSpec is not nil
+// CentralDBEnabled returns true if central db should be created
```

Comment:
Let's add a todo for ourselves to not forget to modify the logic in the future release.
```suggestion
// CentralDBEnabled returns true if central db should be created
// TODO(ROX-13557): modify the logic to enable Postgres DB by default.
```